cpp/src/neighbors/faiss_select/Comparators.cuh

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
I totally agree that we should move to our own select-k in a future release,   probably at the same time as the updating the fusedL2KNN code (which we are tracking in #125).  

Sorry if I wasn't clear above there: I was trying to say that I'd like to keep the faiss_select code in raft instead of moving to cuvs, since we eventually should stop using faiss_select - rather than we should stop using faiss_select in this PR.

Reply:
Oh I see what you mean. I think we can do that for sure.

Reply:
thanks! I've updated #125 to also mention updating RBC - (and I might take this on for the 24.10 release if I can find a bit of time then)